api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +40,13 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * @see SchedulerService#getTaskByUuid(String)
```

Comment:
Can you include the full package name for SchedulerService?

Reply:
Same for String

Reply:
i have fixed it